doc/api/http.md

```diff
@@ -917,6 +917,48 @@ Example:
 var contentType = response.getHeader('content-type');
 ```
 
+### response.getHeaderNames()
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {Array}
+
+Returns an array containing the names of the current outgoing headers. All
+header names are lowercase.
+
```

Comment:
optional nit: might be worthwhile to indicate whether these are sorted in any way or appear in insertion-order